README.md

```diff
@@ -116,6 +116,7 @@ Since version 2.2, this plugin exposes a couple of `hyprctl` commands. That is,
 | Command | description | args | example|
 |------------|-------------|------|--------|
 | printdesk (vdesk)| Prints to Hyprland log the specified vdesk or the currently active vdesk* (if no argument is given) | optional vdesk, see [above](#hyprctl-dispatchers) | `hyprctl printdesk` or `hyprctl printdesk 2` or `hyprctl printdesk coding`|
+| printdesks | Prints information about all vdesks | `none` | `hyprctl printdesks` |
```

Comment:
Forgot to mention this in the first round. Would you mind changing the name of the command? I feel it's too similar to the other one and could easily generate confusion

Reply:
I'm not very imaginative when it comes to names. I now renamed the command to `printstate`. If you have any better name for the command let me know